spdm_dump/spdm/spdm_dump_spdm.c

```diff
@@ -968,6 +968,11 @@ void dump_spdm_certificate(const void *buffer, size_t buffer_size)
         return;
     }
 
+    if (spdm_response->header.param1 >= SPDM_MAX_SLOT_COUNT) {
+        printf("spdm_response->header.param1 is not right\n");
+        return;
+    }
+
```

Comment:
This is not right.
The first step is to dump all information.

Then do the check before refer to the data.